### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by hack-e-d**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/vijay.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Vijay's profile",
+    "repo": "https://github.com/hack-e-d/profile.github.io",
+    "owner": {
+        "username": "hack-e-d",
+        "email": "pvvijayraj1998@gmail.com",
+    },
+    "record": {
+        "CNAME": "hack-e-d.github.io/profile/"
```
Comment (MouseMoosz) [inline domains/vijay.json:9]: `CNAME` record cannot contain any slashes.
Check the [docs](https://docs.is-a.dev/github_pages/) for information about setting up a subdomain from is-a.dev with GitHub pages.

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/vijay.json (hunk 1 above)
Comment (hack-e-d) [reply domains/vijay.json]: changed with slash removal

File: domains/vijay.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Vijay's profile",
+    "repo": "https://github.com/hack-e-d/profile.github.io",
+    "owner": {
+        "username": "hack-e-d",
+        "email": "pvvijayraj1998@gmail.com",
+    },
+    "record": {
+        "CNAME": "hack-e-d.github.io"
```
Comment (VaibhavSys) [inline domains/vijay.json:9]: That shows invalid GitHub site, and the repository is not found either. Maintainers cannot verify the site's content without those.

**Review by VaibhavSys — CHANGES_REQUESTED** [review]

File: domains/vijay.json (hunk 2 above)
Comment (hack-e-d) [reply domains/vijay.json:9]: Only if I add the /profile dose the site load so how do I mention the link ?

Comment (VaibhavSys) [reply domains/vijay.json:9]: > Only if I add the /profile dose the site load so how do I mention the link ?

You can change the name of the repository to "<YOUR_USERNAME>.github.io" but it isn't required.
https://hack-e-d.github.io/profile/ is your website.

**Review by VaibhavSys — APPROVED** [review]

File: domains/vijay.json (hunk 2 above)
Comment (hack-e-d) [reply domains/vijay.json:9]: Yes this is it

Comment (TheBotlyNoob) [inline domains/vijay.json:9]: You can either change the repository name to `hack-e-d.github.io` or change the JSON to:
```suggestion
{
    "description": "Vijay's profile",
    "repo": "https://github.com/hack-e-d/profile.github.io",
    "owner": {
        "username": "hack-e-d",
        "email": "pvvijayraj1998@gmail.com",
    },
    "record": {
        "URL": "https://hack-e-d.github.io/profile"
    }
}
```

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

**Pull request closed without merging**